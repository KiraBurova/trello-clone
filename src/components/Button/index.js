import React from 'react';

import { Base } from './styles/button';

const Button = ({ children }) => {
  return <Base>{children}</Base>;
};

export default Button;
