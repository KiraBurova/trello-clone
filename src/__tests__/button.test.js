import React from 'react';

import { render } from '@testing-library/react';

import Button from '../components/Button';

describe('<Button />', () => {
  it('renders', () => {
    const { container, getByText } = render(<Button>Click me</Button>);

    expect(getByText('Click me')).toBeTruthy();
    expect(container.firstChild).toMatchSnapshot();
  });
});
