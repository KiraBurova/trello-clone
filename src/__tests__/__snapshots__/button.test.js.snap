// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> renders with basic props 1`] = `
<button
  class="button__Base-sc-1caough-0 eWKTrv"
>
  Click me
</button>
`;
